lightcurve/views.py

```diff
@@ -175,3 +177,26 @@ def all_lightcurve(request):
 
     return render_to_response('lightcurve_list.html', {"contexts": context_list, "request_all": True},
             context_instance=RequestContext(request))
+
+
+def download(request, lightcurve_id):
+    file_type = request.GET.get('file_type')
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="LightCurve' + lightcurve_id + '.' + file_type + '"'
```

Comment:
Let's use python string formatters for this
```
... = 'attachment; filename="Lightcurve%d.%s"' % (lightcurve_id, file_type)
```

Reply:
Done.
Although I get type error if I don't use `int(lightcurve_id)`. Should I do `%s.%s` instead? Or `int()`'s fine.